node-gui/src/main_window/main_widget/tabs/wallet/stake.rs

```diff
@@ -61,59 +61,72 @@ const CREATE_STAKING_POOL_TOOLTIP_TEXT: &str = "A staking pool locks the pledge
 const START_STAKING_TOOLTIP_TEXT: &str =
     "If you have created pools, this will activate staking. The node must be kept running, in order to assist the network in creating blocks and earn rewards.";
 
+const DECOMMISSION_POOL_ID_TOOLTIP_TEXT: &str = "The pool ID to decommission.";
+const DECOMMISSION_POOL_ADDRESS_TOOLTIP_TEXT: &str =
```

Comment:
This address has nothing to do with the pool. Calling it POOL_ADDRESS is misleading.

Maybe DECOMMISSION_COINS_DESTINATION_ADDRESS_TOOLTIP_TEXT?